api/jobs_test.go

```diff
@@ -32,11 +32,15 @@ var ctx = context.Background()
 
 func TestCreateJobHandlerWithValidID(t *testing.T) {
 	t.Parallel()
-	NewID = func() string { return testJobID1 }
+	api.NewID = func() string { return testJobID1 }
+
+	jobsCollectionMock := &apiMock.JobStorerMock{
+		CreateJobFunc: func(ctx context.Context, id string) (models.Job, error) { return models.NewJob(id), nil },
+	}
 
 	Convey("Given a Search Reindex Job API that can create valid search reindex jobs and store their details in a map", t, func() {
-		api := Setup(ctx, mux.NewRouter(), &store.DataStore{})
-		createJobHandler := api.CreateJobHandler(ctx)
+		jobStoreApi := api.Setup(ctx, mux.NewRouter(), jobsCollectionMock)
```

Comment:
Not a jobStore Api so this name is misleading. I'd rename, perhaps to `reindexApi` or even `apiInstance` to differentiate it from the api package???

Reply:
NB. I haven't flagged all instances of where this'll change but there are more examples below…

Reply:
fair enough - I've named it apiInstance now